crypto-ffi/src/core_crypto/epoch_observer.rs

```diff
@@ -55,12 +56,19 @@ pub trait EpochObserver: Send + Sync {
 #[cfg(not(target_family = "wasm"))]
 struct ObserverShim(Arc<dyn EpochObserver>);
 
+#[cfg(not(target_family = "wasm"))]
+impl fmt::Debug for ObserverShim {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_tuple("ObserverShim").field(&"Arc<dyn EpochObserver>").finish()
+    }
+}
```

Comment:
This should also be doable via `derive_more` crate, please see https://docs.rs/derive_more/latest/derive_more/derive.Debug.html.